packages/shared/src/telemetry/events/component-mounted.ts

```diff
@@ -2,7 +2,24 @@ import type { TelemetryEventRaw } from '@clerk/types';
 
 const EVENT_COMPONENT_MOUNTED = 'COMPONENT_MOUNTED';
 const EVENT_COMPONENT_OPENED = 'COMPONENT_OPENED';
-const EVENT_SAMPLING_RATE = 0.1;
+const EVENT_SAMPLING_RATE = getComponentSamplingRate();
+
+/**
+ * Get the sampling rate for component mounted events for account portal from NEXT_PUBLIC_CLERK_COMPONENT_ACCOUNT_PORTAL_SAMPLING_RATE environment variable.
+ *
+ * @internal
+ */
+function getComponentSamplingRate(): number {
+  const samplingRate = process.env.NEXT_PUBLIC_CLERK_COMPONENT_ACCOUNT_PORTAL_SAMPLING_RATE;
```

Comment:
Should this be named to describe what it controls, rather than being coupled to where it's used? Like `NEXT_PUBLIC_CLERK_TELEMETRY_COMPONENT_MOUNTED_SAMPLE_RATE` or something?

Reply:
sure